apps/csci32-nextjs-meier/src/app/games/random-number-guesser/page.tsx

```diff
@@ -0,0 +1,279 @@
+'use client'
+import React, { useState, useEffect } from 'react'
+import PageLayout from '@/components/client/PageLayout'
+import { Input } from '@repo/ui/src/input'
+import { Button } from '@repo/ui/src/button'
+import { Size } from '@repo/ui/src/size'
+import { Variant } from '@repo/ui/src/variant'
+import getRandomint from '@math/getRandomInt'
+import gifs from './gifs'
+
+export default function RandomNumberGuesserGame() {
+  // States for game configuration and gameplay
+  const [currentScreen, setCurrentScreen] = useState('welcome')
```

Comment:
I'll approach this code review the way that I would with a coworker, however given that this is academic the code review suggestions and nitpicks are not necessary to implement and only provided to show items that might get called out in the workplace.

It would be interesting to consider having an enum for screens. That would provide autocomplete for a future developer and make sure nobody typos.

```
enum Screens {
  welcome,
  ....
}
```